VersionControlUI/src/DiffManager.cpp

```diff
@@ -276,16 +276,19 @@ bool DiffManager::findChangedNode(Model::TreeManager* treeManager, Model::NodeId
 void DiffManager::createOverlaysForChanges(Visualization::ViewItem* diffViewItem,
 														 QList<ChangeWithNodes> changesWithNodes)
 {
-	static const QString arrowLayer = "move_arrows";
-	diffViewItem->setArrowStyle(arrowLayer, "move_arrow");
+	static const QString moveArrowLayer = "move_arrows";
+	diffViewItem->setArrowStyle(moveArrowLayer, "move_arrow");
+
+	static const QString modifyArrowLayer = "modify_arrows";
+	diffViewItem->setArrowStyle(modifyArrowLayer, "modify_arrow");
 
 	QSet<Visualization::Item*> allItemsToScale;
 	for (auto change : changesWithNodes)
 	{
 		QString highlightOverlayStyle;
 		QString highlightOverlayName;
-		QString arrowIconOverlayStyle;
-		QString arrowIconOverlayName;
+		QString arrowIconOverlayStyle = "";
```

Comment:
instead of assigning "" to the strings here, just leave them uninitialized. Below you can use `isNull()` to check if the strings have changed.